daemon/src/engine/qcommon/q_math.cpp

```diff
@@ -1800,6 +1540,7 @@ void MatrixIdentity( matrix_t m )
 	m[ 15 ] = 1;
 }
 
+// Unused
```

Comment:
Sadly we should keep most of this math stuff as we don't have a replacement and the renderer is very likely to use some of them.